cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/type/PersonDrawWidget.kt

```diff
@@ -0,0 +1,21 @@
+package com.cognifide.cogboard.widget.type
+
+import com.cognifide.cogboard.CogboardConstants
+import com.cognifide.cogboard.widget.BaseWidget
+import io.vertx.core.Vertx
+import io.vertx.core.json.JsonObject
+
+class PersonDrawWidget(vertx: Vertx, config: JsonObject) : BaseWidget(vertx, config) {
+
+    init {
+        super.config.put(CogboardConstants.PROP_SCHEDULE_PERIOD, 1L)
```

Comment:
Is this really needed? if CogboardConstants.PROP_SCHEDULE_PERIOD == 0 then widget will execute only once after save or app restart.